src/mantle/facade/class-request.php

```diff
@@ -10,6 +10,143 @@
+ * @method static void overrideGlobals()
+ * @method static void setTrustedProxies(array $proxies, int $trustedHeaderSet)
+ * @method static string[] getTrustedProxies()
+ * @method static int getTrustedHeaderSet()
+ * @method static void setTrustedHosts(array $hostPatterns)
+ * @method static string[] getTrustedHosts()
+ * @method static string normalizeQueryString(string|null $qs)
+ * @method static void enableHttpMethodParameterOverride()
+ * @method static bool getHttpMethodParameterOverride()
+ * @method static \Symfony\Component\HttpFoundation\Session\SessionInterface getSession()
+ * @method static bool hasPreviousSession()
+ * @method static bool hasSession(bool $skipIfUninitialized = false)
+ * @method static void setSession(\Symfony\Component\HttpFoundation\Session\SessionInterface $session)
+ * @method static array getClientIps()
+ * @method static string|null getClientIp()
+ * @method static string getScriptName()
+ * @method static string getPathInfo()
+ * @method static string getBasePath()
+ * @method static string getBaseUrl()
+ * @method static string getScheme()
+ * @method static int|string|null getPort()
+ * @method static string|null getUser()
+ * @method static string|null getPassword()
+ * @method static string|null getUserInfo()
+ * @method static string getHttpHost()
+ * @method static string getRequestUri()
+ * @method static string getSchemeAndHttpHost()
+ * @method static string getUri()
+ * @method static string getUriForPath(string $path)
+ * @method static string getRelativeUriForPath(string $path)
+ * @method static string|null getQueryString()
+ * @method static bool isSecure()
+ * @method static string getHost()
+ * @method static void setMethod(string $method)
+ * @method static string getMethod()
+ * @method static string getRealMethod()
+ * @method static string|null getMimeType(string $format)
+ * @method static string[] getMimeTypes(string $format)
+ * @method static string|null getFormat(string|null $mimeType)
+ * @method static void setFormat(string|null $format, string|string[] $mimeTypes)
+ * @method static string|null getRequestFormat(string|null $default = 'html')
+ * @method static void setRequestFormat(string|null $format)
+ * @method static string|null getContentTypeFormat()
+ * @method static void setDefaultLocale(string $locale)
+ * @method static string getDefaultLocale()
+ * @method static void setLocale(string $locale)
+ * @method static string getLocale()
+ * @method static bool isMethod(string $method)
+ * @method static bool isMethodSafe()
+ * @method static bool isMethodIdempotent()
+ * @method static bool isMethodCacheable()
+ * @method static string|null getProtocolVersion()
+ * @method static string|resource getContent(bool $asResource = false)
+ * @method static \Symfony\Component\HttpFoundation\InputBag getPayload()
+ * @method static array toArray()
```

Comment:
is this different from to_array on line 40?

Reply:
yes, it has both